data_structures/stack/infixToPostfix.c

```diff
@@ -0,0 +1,100 @@
+#include<stdio.h>
+#include<string.h>
+
+struct Stack
+{
+    int top;
+    unsigned capacity;
+    int* array;
```

Comment:
I think ``array`` should be of type ``char`` because it is used to store operators (which are characters) to avoid unnecessary implicit type conversion in ``pop`` and ``push``.